go.mod

```diff
@@ -96,6 +96,7 @@ replace (
 	github.com/onsi/gomega => github.com/onsi/gomega v1.14.0
 	github.com/openshift/api => github.com/openshift/api v0.0.0-20220929131120-2bde012f248a
 	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20200325131901-f7baeb993edb
+	golang.org/x/net => golang.org/x/net v0.4.0
```

Comment:
Isn't this an old version of /x/net?

Reply:
Hi @jgwest , I realised there was a typo in the version.

Thank you for catching this and correcting it